dev/README.md

```diff
@@ -42,11 +42,11 @@ Here is a list of the **required** environment variable:
 
 ### Configuration via "localconfig" file
 
-If a file named `localconfig` is created in the [dev/bin](bin) folder, all its content is loaded before every script.
+If a file named `localconfig` is created in the [dev/config](bin) folder, all its content is loaded before every script.
```

Comment:
you should change the name of the directory also in the link destination, otherwise it will be inconsistent (`[dev/config](config)`)

Reply:
I need to update the whole doc with this new script :)

Reply:
doc updated